srv/modules/runners/bootstrap.py

```diff
@@ -108,8 +108,8 @@ def core(non_interactive=False):
 
 def ceph(non_interactive=False):
     # TODO: improve return messages
-    if not initialize(non_interactive=non_interactive):
-        return False
+    # if not initialize(non_interactive=non_interactive):
```

Comment:
This shouldn't have been commented. I just comment this because `setup` takes too long for development cycles. I'll push an uncommented version of this file